src/components/PageNotFound/PageNotFound.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './pageNotFound.css';
+import { useHistory } from 'react-router-dom';
+
+const PageNotFound = () => {
+  const history = useHistory();
+  return (
+    <div className="container">
+      <div className="header-bar">
+        <strong>netflix</strong>
+        <span>roulette</span>
+      </div>
```

Comment:
i think it's a common component, pls move it into separate component if it's used more then 1 times
 if i wrong you will skip this component